src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -54,18 +54,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             // Mount the server-side Blazor app on /subdir
             app.Map("/subdir", subdirApp =>
             {
-                // The following two lines are equivalent to:
-                //     endpoints.MapComponentsHub<Index>();
-                //
-                // However it's expressed using routing as a way of checking that
-                // we're not relying on any extra magic inside MapComponentsHub, since it's
-                // important that people can set up these bits of middleware manually (e.g., to
-                // swap in UseAzureSignalR instead of UseSignalR).
```

Comment:
Comment removed because we're going the other way. We're providing all of the things that are meaningful and returning a type that implements the same builder API as SignalR.